apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -91,9 +95,11 @@ public static void premain(String agentArgs, Instrumentation instrumentation) th
             return;
         }
 
-        final ByteBuddy byteBuddy = new ByteBuddy().with(TypeValidation.of(Config.Agent.IS_OPEN_DEBUGGING_CLASS));
+        LOGGER.info("Skywalking agent begin to install transformer ...");
+        String nameTrait = getNameTrait();
+        DelegateNamingResolver.setNameTrait(nameTrait);
```

Comment:
These two should move inside `newAgentBuilder`. And `"sw$"` should be moved into `Constants`. And `SWNativeMethodStrategy` doesn't need to forward this, `DelegateNamingResolver#NAME_TRAIT` should be removed too.
Let's reduce those unnecessary codes.